### PR TITLE
updated release guide

### PR DESCRIPTION
**Pull request by orhovy**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Screenshots
Paste here any images that will help the reviewer

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by bakatzir — APPROVED** [review]

**Review by yaakovi — APPROVED** [review]

File: docs/release_guide.md [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
-8) Click on **Draft a new release**.
-9) Update the **Tag version** and **Release title** - the form should be `vX.X.X` .
-10) In the **Describe the release** text box enter the CHANGELOG contents for this release.
-11) If nightly passes ok, press **Publish release** - your release is now public.
-12) Update the [**dev-requirements-py3.txt**](https://github.com/demisto/content/blob/master/dev-requirements-py3.txt) file again, this time with the release tag.
-13) Run the regular build again or force merge your PR to the Content repository.
-14) Announce regarding the SDK release in the content-team slack channel.
-15) Update **CHANGELOG.md** file - create header with the release version instead of `# Changelog` in the format `# X.X.X` e.g. `# 1.0.0`.
+6) Update the version of the SDK in Demisto's Content repository by updating the demisto-sdk version in the [**dev-requirements-py3.txt**](https://github.com/demisto/content/blob/master/dev-requirements-py3.txt) file. Use the release branch first - replace the `demisto-sdk==version` line with this line: `git+https://github.com/demisto/demisto-sdk.git@release-branch-name.`
+
```
Comment (roysagi) [inline docs/release_guide.md:16]: @anara123 we should consider adding a similar step to 6 for the private repo

File: docs/release_guide.md [hunk 2]
```diff
@@ -1,28 +1,28 @@
...
-  Make sure both builds are green before releasing.
-5) Update the version of the SDK in Demisto's Content repository by updating the demisto-sdk version in the [**dev-requirements-py3.txt**](https://github.com/demisto/content/blob/master/dev-requirements-py3.txt) file. Use the release branch first - replace the `demisto-sdk==version` line with this line: `git+https://github.com/demisto/demisto-sdk.git@release-branch-name.`
-6) If there was any change in the content creation steps (**create content artifacts** or **unify** commands) - we need to check that the new content is valid.
+### Validation before release:
+1) Make sure the **CHANGELOG.md** file is in order and is updated with all the changes in the current release.
+2) In demisto-sdk, create a new release branch, named after the release version in the format `X.X.X`, e.g. `1.0.0`
+3) Make sure that sdk-nightly and sdk-master builds are both green. If there were no new commits to the sdk after
+ the nightly tests finished you can skip step 4.
+4) If there were new commits to the sdk after the nightly tests finished you will need to trigger sdk nightly build again in order to test sdk master on content branch
```
Comment (reutshal) [inline docs/release_guide.md:8]:
```suggestion
4) If there were new commits to the sdk after the nightly tests finished you will need to trigger sdk nightly build again in order to test sdk master on content branch\
```
Add this if you want a line break

**Review by reutshal — APPROVED** [review]
Looks good.
Maybe it is worth mentioning that the circle token is private and should be generated for each user, It raised questions every time one was trying to use the scripts.

File: docs/release_guide.md (hunk 1 above)
Comment (anara123) [reply docs/release_guide.md:16]: Correct. We should. But we should define which tests should run when we open such pr

**Pull request merged**